packages/utils/src/string.ts

```diff
@@ -97,7 +97,7 @@ export function isMatchingPattern(value: string, pattern: RegExp | string): bool
     return pattern.test(value);
   }
   if (typeof pattern === 'string') {
-    return value.indexOf(pattern) !== -1;
+    return value.includes(pattern);
```

Comment:
doesn't work in IE11 - and we don't want folks to require the polyfill even when they downcompile

Reply:
Interesting. I saw some usage of `String.includes()` elsewhere in the SDK. I thought it was fair game.